__tests__/acceptance/eslint-with-todo-formatter-test.ts

```diff
@@ -1238,4 +1251,52 @@ describe('eslint with todo formatter', function () {
       }
     });
   }
+
+  it('when given FORMAT_TODO_AS will output with that formatters format', async () => {
+    project.write({
+      src: {
+        'with-errors-0.js': getStringFixture('with-errors-0.js'),
+      },
+    });
+
+    const result = await runBin({
+      env: {
+        FORMAT_TODO_AS: '@microsoft/eslint-formatter-sarif',
+      },
+    });
+
+    expect(getMessagesFromSarif(JSON.parse(result.stdout), ['error'])).toEqual([
```

Comment:
Can we use `toHaveLength` instead, as the content of the message isn't super relevant to the test.